app/assets/v2/js/pages/townsquare.js

```diff
@@ -42,8 +42,9 @@ $(document).ready(function() {
 
   var get_redir_location = function(tab) {
     let trending = $('#trending').is(':checked') ? 1 : 0;
+    let personal = $('#personal').is(':checked') ? 1 : 0;
 
-    return '/?tab=' + tab + '&trending=' + trending;
+    return '/?tab=' + tab + '&trending=' + trending + '&personal=' + personal;
```

Comment:
Shouldn't be good to use string literals here?

Reply:
I think we can remove this as we are making personal feed a separate tab now.